cucumber/authenticators_jwt/features/authn_jwt_token_schema.feature

```diff
@@ -448,4 +448,264 @@ Feature: JWT Authenticator - Token Schema
+    And I successfully set authn-jwt "enforced-claims" variable to value "claim.name, claim.ant..., claim_name"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "claim.name": "claim.name.value",
+      "claim.ant...": "claim.ant...value",
+      "claim_name": "claim_name_value"
+    }
+    """
+    And I save my place in the audit log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 200
+    And The following appears in the log after my savepoint:
+    """
+    cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
+    """
 
+  Scenario Outline: ONYX-10873 - Broken claims mapping - 401 Error
+    Given I extend the policy with:
+    """
+    - !variable conjur/authn-jwt/raw/mapping-claims
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/sub: mysub
+        authn-jwt/raw/ref: myref
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I successfully set authn-jwt "mapping-claims" variable to value "<mapping>"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "sub":"mysub",
+      "ref":"mybranch"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    <err>
+    """
+    Examples:
+      | mapping                     | err                                                                             |
+      |   branch: ref, branch:sub   | Failed to parse mapping claims, annotation name value 'branch' appears twice or more |
+      |   branch: sub, job: sub     | Failed to parse mapping claims, claim name value 'sub' appears twice or more    |
+
+  Scenario Outline: ONYX-10875 - Standard claim in mapping - 401 Error
+    Given I extend the policy with:
+    """
+    - !variable conjur/authn-jwt/raw/mapping-claims
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/sub: mysub
+        authn-jwt/raw/ref: myref
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I successfully set authn-jwt "mapping-claims" variable to value "<mapping>"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "sub":"mysub",
+      "ref":"mybranch"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    CONJ00105E Failed to validate claim, claim name 'exp' is in denylist '["iss", "exp", "nbf", "iat", "jti", "aud"]'>.>
+    """
+    Examples:
+      | mapping        |
+      |   branch: exp  |
+      |   exp: sub     |
```

Comment:
Why we have the same error ? this error related to annotation no?

Reply:
But in both of them we deal with standard claim. Same class checking annotation and claim because in the end its a strange thing

Reply:
Do you think it should be diffrerent errors for  annotation and claim. If so a ticket out of scope of this PR should be opened because its a test PR .

@hilagross @shulifink @InbalZilberman @eladkug

Reply:
I agree with @eladkug

Reply:
Talked about this with @hilagross you are right @eladkug .

@sashaCher can you take it in the tests PR ? Or should we open another ticket for this log change?